core/src/main/java/com/orange/ouds/core/component/samples/OudsColoredBoxSample.kt

```diff
@@ -17,6 +17,9 @@ import androidx.compose.runtime.Composable
 import com.orange.ouds.core.component.coloredbox.OudsColoredBox
 import com.orange.ouds.theme.tokens.OudsColorKeyToken
 
+/**
+ * @suppress
+ */
 @Composable
 fun SimpleOudsColoredBox() {
```

Comment:
We can also set this method `internal` instead  of using `@suppress`. This is probably even better because it won't pollute the autocompletion.